playout/moggy.c

```diff
@@ -969,29 +970,27 @@ playout_moggy_assess(struct playout_policy *p, struct prior_map *map, int games)
 	} foreach_free_point_end;
 }
 
+
+#define permit_move(c)  playout_permit(p, b, c, m->color)
```

Comment:
I'm a bit at loss here - our permit hook calls playout_permit() which calls playout_permit_move() which calls our permit hook again. Am I wrong to be concerned here?

Maybe it is all connected to the "alt" parameter, which is undocumented and I wonder if it has only one meaning.